src/xdp-context.c

```diff
@@ -207,7 +207,7 @@ export_host_portal_implementation (GDBusConnection        *connection,
    * method calls must see the modified value.
    */
 
-  g_autoptr(GError) error = NULL;
+  g_autoptr (GError) error = NULL;
```

Comment:
GNOME is not consistent on this, but xdg-desktop-portal is: no space in the g_autoptr. I don't know why tbh; everywhere else, we _do_ use a space before opening parentheses....